app/modules/site_settings/resources.py

```diff
@@ -515,6 +515,21 @@ def get(self):
         }
 
 
+@api.route('/public-data')
+class PublicData(Resource):
+    def get(self):
+
+        from app.modules.individuals.models import Individual
+        from app.modules.sightings.models import Sighting
+        from app.modules.users.models import User
+
+        return {
+            'num_users': User.query_search().count(),
```

Comment:
Using `cls.query_search()` without arguments is identical to using `cls.query`.  I don't have any opposition to using `query_search`, especially if we want to add search terms down the line, but `cls.query.count()` is a simpler alternative.  I am putting this note here for documentation, not really suggesting a change

Reply:
One thing I wanted to ask you about this is is there any way that we can do a query_search excluding users with role internal and role staff?
This is requested functionality but because these are enumerated values, I am thinking that the only way will be 
read all users
filter out (using list comprehension) all users with role internal or staff.

What I'd be keen to know is is there a more efficient way to do this using an ES query?